src/sdg/producten/models/product.py

```diff
@@ -79,6 +84,52 @@ def upn_uri(self):
     def upn_label(self):
         return self.upn.upn_label
 
+    @property
+    def upn_is_verweiderd(self):
+        return self.upn.is_verwijderd
+
+    @property
+    def is_sdg_product(self):
+        return bool(self.upn.sdg)
+
+    @cached_property
+    def referentie_product_publicatie_datum(self):
```

Comment:
Typically start boolean fields with "is_" or "has_"

```suggestion
    def has_referentie_product_publicatie_datum(self):
```